ilastik/ilastik_logging/default_config.py

```diff
@@ -218,7 +218,14 @@ def get_default_config(
     return default_log_config
 
 
+LOGGING_CONFIGURED = False
+
+
 def init(format_prefix="", output_mode=OutputMode.LOGFILE_WITH_CONSOLE_ERRORS, logfile_path=DEFAULT_LOGFILE_PATH):
+    global LOGGING_CONFIGURED
```

Comment:
Maybe this is an overkill, but there is a way to avoid the global variable:

```python
def call_once(func):
    def wrapper(*args, **kwargs):
        if not wrapper.called_once:
            func(*args, **kwargs)
            wrapper.called_once = True

    wrapper.called_once = False
    functools.update_wrapper(wrapper, func)
    return wrapper


@call_once
def init(...):
    ...
```

Reply:
yeah I found something very similar on stackoverflow, but indeed thought it was overkill, but maybe I'll make the global variable private...